source/_integrations/dsmr.markdown

```diff
@@ -46,10 +46,20 @@ DIY solutions (ESP8266 based):
 
 - [esp8266_p1meter (fliphess)](https://github.com/fliphess/esp8266_p1meter)
 
+## Configuration
+
+There are two ways to integrate your DSMR into Home Assistant
+
+### Via the frontend
+
+Menu: **Configuration** -> **Integrations**. Search for "DSMR Slimme Meter" and follow the configuration flow.
+
+### Via the configuration file
+
 ```yaml
 # Example configuration.yaml entry
-sensor:
-  - platform: dsmr
+dsmr:
+  port: /dev/ttyUSB0
```

Comment:
We should not introduce new YAML options for integrations that communicate with a device or server.

For more information see ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

Reply:
Yeah, I get your point. The idea behind it was to go easy on people who currently use yaml and have an easy way to rewrite the configuration to a domain (not introduce new options).

I have no problems deprecating yaml alltogether and only allow UI configuration.

Reply:
I have to say at the moment I made this I took examples from the docs and core and saw this as a nice solution in line with existing integrations, but I guess there is a lot of legacy around especially as ADR-0010 is not that old.

I'll remove yaml support from the docs and from the core PR that is linked to this PR